### DISCUSSION TITLE
Confirm Dialog change and localize buttons text

### DISCUSSION THREAD

**Discussion by luis-Applicationcare** [primefaces/community#60] [PrimeVue] [answered]
Hello,

Is it possible to change the text of the "Yes" "No" buttons in a simple way? I checked the documentation and I can't find the way.

Thanks

**Comment by tugcekucukoglu** [comment, marked as answer]
Hello,

For ConfirmDialog to work, the `require` method of the `$confirm` instance that does all the configuration must be called. This is where the button labels can be changed. According to your question, these are `acceptLabel` and `rejectLabel`. 

```
this.$confirm.require({
       message: 'Are you sure you want to proceed?',
       acceptLabel: '***',
       rejectLabel: '***'
});
```


If you are already using PrimeVue Locale, just change the `accept` and `reject` options.
